Alarmi/Alarmi/Sources/Global/Extensions/DateComponent+.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  DateComponent+.swift
+//  Alarmi
+//
+//  Created by Woody on 2022/07/31.
+//  Copyright © 2022 MoTe. All rights reserved.
+//
+
+import Foundation
+
+extension DateComponents {
+    static func koreaHour(_ hour: Int) -> Self {
+        return .init(
+            timeZone: TimeZone(identifier: "Asia/Seoul"),
+            hour: hour
+        )
+    }
+}
```

Comment:
[제가 작성했던 코드](https://github.com/DeveloperAcademy-POSTECH/MC3-Team7-MoTe/pull/84/files)에서는 한국 시간을 변환할 때 사용하는 DateComponent에 년월일도 넣었었는데, 거기엔 이유가 있었습니다.
시뮬레이터 상의 오류인지는 모르겠지만 년월일을 넣지 않으면 아래 사진과 같이 잘못된 시간이 반환되게 됩니다. 그런데 년월일을 현재 날짜로 넣게 되면 제대로 된 시간이 나오게 됩니다.
이 이유가 아니더라도 외국은 서머타임제가 있는 경우도 있기 때문에, 어플을 킨 날짜에 해당하는 시간을 반환해야 한다고 생각합니다.

- 현재 상태
한국 7시가 베를린 23:25로 나옴. (틀린 시간)
<img width="567" alt="image" src="https://user-images.githubusercontent.com/75792767/182139722-b1d710a8-c779-49a7-9f8a-edddc3be5362.png">

- DateComponent에 year, month, day를 추가했을 때
한국 7시가 베를린 0시로 나옴. (맞는 시간)
<img width="556" alt="image" src="https://user-images.githubusercontent.com/75792767/182140021-e77129a4-bd96-4d76-8d11-6f84985ede4c.png">

```suggestion
extension DateComponents {
    static func koreaHour(_ hour: Int) -> Self {
        return .init(
            timeZone: TimeZone(identifier: "Asia/Seoul"),
            year: Calendar.current.component(.year, from: Date()),
            month: Calendar.current.component(.month, from: Date()),
            day: Calendar.current.component(.day, from: Date()),
            hour: hour
        )
    }
}
```

Reply:
해당부분